uber/automated_emails.py

```diff
@@ -227,6 +227,9 @@ def days_before(days, dt, until=None):
               lambda a: c.SHIFTS_CREATED and days_before(5, c.UBER_TAKEDOWN)
                                          and a.ribbon == c.VOLUNTEER_RIBBON and a.takes_shifts and a.weighted_hours == 0)
 
+StopsEmail('Your {EVENT_NAME} shift schedule', 'shifts/schedule.html',
+           lambda a: c.SHIFTS_CREATED and days_before(1, c.UBER_TAKEDOWN) and a.weighted_hours)
+
```

Comment:
I made this email get sent out the day before "uber takedown" so that people would have one last chance to change their shifts.  Dunno if we want to keep it that way, we'll see how that works out at Classic.